idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -179,86 +182,89 @@ def build(self):
+            "property_package_args":
+                self.config.cold_side.property_package_args})
 
-        self.cold_side.add_state_blocks(has_phase_equilibrium=False)
+        self.cold_fluid.add_state_blocks(has_phase_equilibrium=False)
 
-        self.cold_side.add_material_balances(
+        self.cold_fluid.add_material_balances(
             balance_type=MaterialBalanceType.componentTotal,
             has_mass_transfer=False,
             has_phase_equilibrium=False,
             has_rate_reactions=False)
 
-        self.cold_side.add_momentum_balances(
+        self.cold_fluid.add_momentum_balances(
             balance_type=MomentumBalanceType.pressureTotal,
             has_pressure_change=True)
 
-        # ======================================================================
+        # =====================================================================
         # Add Ports to control volumes
         # hot-side
         self.add_inlet_port(name="hot_inlet",
-                            block=self.hot_side, doc='inlet Port')
+                            block=self.hot_fluid, doc='inlet Port')
         self.add_outlet_port(name="hot_outlet",
-                             block=self.hot_side, doc='outlet Port')
+                             block=self.hot_fluid, doc='outlet Port')
 
         # cold-side
         self.add_inlet_port(name="cold_inlet",
-                            block=self.cold_side, doc='inlet Port')
+                            block=self.cold_fluid, doc='inlet Port')
         self.add_outlet_port(name="cold_outlet",
-                             block=self.cold_side, doc='outlet Port')
-        # ======================================================================
+                             block=self.cold_fluid, doc='outlet Port')
+        # =====================================================================
         # Add performace equation method
         self._make_params()
         self._make_performance_method()
 
     def _make_params(self):
         self.P = Param(initialize=self.config.passes,
-                       units=None,
+                       units=pyunits.dimensionless,
                        doc="Total number of passes for hot or cold fluid")
         self.PH = RangeSet(self.P,
                            doc="Set of hot fluid passes")
         self.PC = RangeSet(self.P,
                            doc="Set of cold fluid passes(equal to PH)")
 
-        self.plate_thermal_cond = Param(mutable=True,
-                                        initialize=self.config.plate_thermal_cond,
-                                        units=pyunits.W / pyunits.m / pyunits.K,
-                                        doc="Plate thermal conductivity")
+        self.plate_thermal_cond = Param(
+            mutable=True,
+            initialize=self.config.plate_thermal_cond,
+            units=pyunits.W / pyunits.m / pyunits.K,
```

Comment:
It would be nice if these units were not hard coded and depended on the property package instead.